src/config.ts

```diff
@@ -36,6 +36,7 @@ export interface Config {
   forceCleanNpmInstall: boolean;
   csvFileStats: string;
   csvFileRaw: string;
+  partition?: 'measurement';
```

Comment:
What do you think about actually using `'none'` as the default value instead of treating `undefined` as doing no partition?

Reply:
Good call! Will do!